_build-and-test-all.sh

```diff
@@ -13,11 +13,24 @@ ${dockercdc}Up
 
 ./gradlew -x :end-to-end-tests:test build
 
+#Testing db cli
+if [ "${DATABASE}" == "mysql" ]; then
+  echo 'show databases;' | ./mysql-cli.sh -i
+elif [ "${DATABASE}" == "postgres" ]; then
+  echo '\l' | ./postgres-cli.sh -i
+else
+  echo "Unknown Database"
+  exit
```

Comment:
`exit` won't signal an error (to CircleCI) - the build will succeed. It must be `exit 99`